data/autoyast_sle15/autoyast_disk_as_md_member.xml

```diff
@@ -0,0 +1,121 @@
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email/>
+        <reg_code>{{SCC_REGCODE}}</reg_code>
+        <install_updates config:type="boolean">true</install_updates>
+        <reg_server>{{SCC_URL}}</reg_server>
+        <addons config:type="list">
+            <addon>
+                <name>sle-module-server-applications</name>
+                <version>{{VERSION}}</version>
+                <arch>{{ARCH}}</arch>
+            </addon>
+        </addons>
+    </suse_register>
+    <bootloader>
+        <global>
+            <timeout config:type="integer">-1</timeout>
+        </global>
+    </bootloader>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+    <software>
+        <products config:type="list">
+            <product>SLES</product>
+        </products>
+    </software>
+    <users config:type="list">
+        <user>
+            <fullname>Bernhard M. Wiedemann</fullname>
+            <encrypted config:type="boolean">false</encrypted>
+            <user_password>nots3cr3t</user_password>
+            <username>bernhard</username>
+        </user>
+        <user>
+            <encrypted config:type="boolean">false</encrypted>
+            <user_password>nots3cr3t</user_password>
+            <username>root</username>
+        </user>
+    </users>
+    <report>
+        <errors>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </errors>
+        <messages>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </messages>
+        <warnings>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </warnings>
+        <yesno_messages>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </yesno_messages>
+    </report>
+    <partitioning config:type="list">
+        <drive>
+            <type config:type="symbol">CT_DISK</type>
+            <device>/dev/sda</device>
+            <disklabel>gpt</disklabel>
+            <use>all</use>
+            <partitions config:type="list">
+                <partition>
+                    <create config:type="boolean">true</create>
+                    <partition_id config:type="integer">263</partition_id>
+                    <size>8MiB</size>
+                </partition>
+                <partition>
+                    <create config:type="boolean">true</create>
+                    <mount>/boot</mount>
```

Comment:
Shall we implement feature to inject values from `test_data` structure, similarly to what we've done with vars.json?

Reply:
I never thought about that, but it might be great to have the feature I guess, as it will allow to change some data just in one place, rather then check all the places where it is used.

But, I guess it should be done in scope of another task?

Reply:
Sure. Just wanted to hear your opinion. I will create a ticket for that.